pages/linux/switch_root.md

```diff
@@ -0,0 +1,12 @@
+# switch_root
+
+> Use a different filesystem as the root of the mount tree.
+> More information: <https://manned.org/switch_root.8>.
+
+- Move `/proc`, `/dev`, `/sys` and `/run` to the specified filesystem, use this filesystem as the new root and start the specified init process:
+
+`switch_root {{new_root}} {{init_process}}`
+
```

Comment:
```suggestion

- Bind-mount a directory to turn it into a mount point:

`mount --bind {{path/to/directory}} {{path/to/directory}}`
```

Reply:
I would omit this since it is a different command. In general I think that workarounds should not be part of tldr pages, as they are not necessary in basic usage. However, we can keep the note above.